datalad_ria/tests/test_fixtures.py

```diff
@@ -22,3 +22,9 @@ def test_riaserver_fixture(ria_sshserver):
     # base url and local path
     assert len(ria_sshserver) == 2
     assert ria_sshserver[0].startswith('ria+ssh://')
+
+
+def test_common_init_opts_fixture(common_init_opts):
+    assert common_init_opts == \
+           ["encryption=none", "type=external", "externaltype=ora",
+            "autoenable=true"]
```

Comment:
I would consider it more sensible and more resilient to limit the testing to less than all of it. It might make sense to test for `"externaltype=ora" in common_init_opts` to make sure this is initing the correct remote implementation. I am less sure about auto-enable, but it would make sense. Certainly the order is irrelevant, and should not be necessary to be kept.

Reply:
done, thx